src/main/java/Duke.java

```diff
@@ -1,10 +1,203 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LOGO = System.lineSeparator()
+            + "    ___   ______   ________  _______      ___   ____  _____   .--." + System.lineSeparator()
+            + "  .'   `.|_   _ \\ |_   __  ||_   __ \\   .'   `.|_   \\|_   _|.'_\\/_'." + System.lineSeparator()
+            + " /  .-.  \\ | |_) |  | |_ \\_|  | |__) | /  .-.  \\ |   \\ | |  '. /\\,.'" + System.lineSeparator()
+            + " | |   | | |  __'.  |  _| _   |  __ /  | |   | | | |\\ \\| |    \"||\"" + System.lineSeparator()
+            + " \\  `-'  /_| |__) |_| |__/ | _| |  \\ \\_\\  `-'  /_| |_\\   |_    || /\\`" + System.lineSeparator()
+            + "  `.___.'|_______/|________||____| |___|`.___.'|_____|\\____|/\\ ||//\\)" + System.lineSeparator()
+            + "                                                           (/\\\\||/" + System.lineSeparator()
+            + "^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\\||^^^^";
+
+    private static final String HORIZONTAL_BAR = "__________________"
+            + "__________________"
```

Comment:
I think you can put these in the same line, and if you want to keep track of the number of characters, you could add that as a comment